README.md

```diff
@@ -1 +1,179 @@
-# Glaciers
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <br/>
+  <h3><b>Tropical Gateways</b></h3>
+ <br/>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [ Walkthrougth](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Tropical_Gateways] <a name="about-project"></a>
+
+**[Tropical_Gateways]** is a web application for the first capstone project of the first module, it emulates a travel agency based on a template web we were given.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+In this project we used HTML CSS and Javascript
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Mobile_responsive]**
+- **[Flexbox]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [ ] As required by the project requirements, please kindly makes use of GitHub Pages to deploy the web pages and add a link to the deployed project in your README file.